### PR TITLE
Adds initial version of stackable bcrypt tool

### PR DESCRIPTION
**Pull request by soenkeliebau**

### CODE REVIEW

File: stackable-bcrypt/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+use clap::Parser;
+use std::process::exit;
+
+#[derive(Parser)]
+#[clap(
+    about = "bcrypt commandline tool",
+    long_about = "A tiny command line tool to hash a given string using the bcrypt algorithm.\
+        Default cost used is 10, but can be configured. \
+        The output is only the hash, so that this can be used for automation purposes without any parsing.",
+    author = "Stackable GmbH - info@stackable.de"
```
Comment (teozkr) [inline stackable-bcrypt/src/main.rs:10]:
```suggestion
    author
```

Clap can automagically pull this in from Cargo.toml.

File: stackable-bcrypt/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+)]
+struct Opts {
+    #[clap(short, long, default_value = "10")]
+    cost: u8,
+    #[clap(short, long)]
+    input: String,
```
Comment (teozkr) [inline stackable-bcrypt/src/main.rs:16]: Can we take this from stdin instead? argv is world-readable from /proc for the lifetime of the command.

Comment (teozkr) [reply stackable-bcrypt/src/main.rs:16]: That wouldn't help, bash still resolves substitutions before handing off to the program. What would be an alternative would be to have the program take the path to read from as an argument.

Comment (soenkeliebau) [reply stackable-bcrypt/src/main.rs]: reads from stdin now

**Review by teozkr — APPROVED** [review]

**Pull request merged**